src/codegen/mod.rs

```diff
@@ -1270,10 +1344,24 @@ impl CodeGenerator for Enum {
 
         let mut builder = aster::AstBuilder::new().item().pub_();
 
+        let is_bitfield = {
+            ctx.options().bitfield_enums.matches(&name) ||
+            (enum_ty.name().is_none() &&
+             self.variants()
+                .iter()
+                .any(|v| ctx.options().bitfield_enums.matches(&v.name())))
+        };
```

Comment:
We should factor this out to a method on the context or on the `Enum` which takes a context as a parameter. No need for the logic of determining whether a type should be a bitfield to live inside code generation.